src/modules/dream/test/CuboidTest.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2022 European Spallation Source, ERIC. See LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+//===----------------------------------------------------------------------===//
+
+#include <common/testutils/TestBase.h>
+#include <dream/geometry/Cuboid.h>
+#include <dream/readout/DataParser.h>
+
+using namespace Dream;
+
+class CDTGeometryTest : public TestBase {
+protected:
+  DataParser::DreamReadout Readout{0, 0, 0, 0, 0, 0, 0, 0, 0};
+  Config::ModuleParms Parms{false, Config::ModuleType::HR, {0}, {0}};
+  Cuboid geometry;
+  void SetUp() override {}
+  void TearDown() override {}
+};
+
+
+TEST_F(CDTGeometryTest, Rotate0) {
+  int x = 1;
+  int y = 12;
+  geometry.rotateXY(x, y, 0);
+  ASSERT_EQ(x,  1);
+  ASSERT_EQ(y, 12);
+}
+
+TEST_F(CDTGeometryTest, Rotate1) {
+  int x = 1;
+  int y = 12;
+  geometry.rotateXY(x, y, 1);
+  ASSERT_NE(x,  1);
+  ASSERT_NE(y, 12);
```

Comment:
Does ASSERT_NE check thoroughly enough? Can we calculate what the rotated values should be and use ASSERT_EQ instead?